src/core/eme/utils/init_data_store.ts

```diff
@@ -183,7 +183,7 @@ export default class InitDataStore<T> {
   ) : number {
     // Begin by the last element as we usually re-encounter the last stored
     // initData sooner than the first one.
-    for (let i = this._storage.length - 1; i >= 0; i++) {
```

Comment:
:o 
Thanks!

Reply:
:O I pushed-force due to a rebase but it removed your commit. Thankfully I saw this.